hub/test/src/testDrivers.js

```diff
@@ -394,7 +390,41 @@ function testGcDriver() {
   })
 }
 
+function testInMemoryDriver() {
```

Comment:
One gotcha we've had in the past is that people will store different kinds of data (with different `content-type`s), and expect to get the same data back.  This test only checks `application/octet-stream` for plain text.  If we keep the in-memory driver, can this test be expanded to include binary data and structured data?

Reply:
Good idea 👍